docs/observability/metrics.md

```diff
@@ -0,0 +1,73 @@
+---
+seotitle: Monitoring your backend application with custom metrics
+seodesc: See how you can monitor your backend application using Prometheus and Encore.
```

Comment:
```suggestion
seodesc: See how you can monitor your backend application using Grafana Cloud and Encore.
```

Reply:
Maybe we should just drop that and say "using Encore"